NOTICE

```diff
@@ -47,6 +44,86 @@ types-urllib3 1.26.15 : Apache License 2.0
 urllib3 1.26.9 : MIT License
 zipp 3.6.0 : MIT License
 
+
+Copyright Text: 
+
+Click - Python Command Line Utility 8.0.4 pypi:click/8.0.4
+	Copyright 2002-2006 Python Software Foundation. All rights reserved
+	Copyright 2001-2006 Gregory P. Ward
+	Copyright 2001-2006 Gregory P. Ward. All rights reserved
+	Copyright 2002-2006 Python Software Foundation
+CommonMark-py 0.9.1 pypi:commonmark/0.9.1
+	Copyright (c) 2014, John MacFarlane All rights reserved
+	Copyright (c) 2014, Bibek Kafle and Roland Shoemaker
+Packaging 21.3 pypi:packaging/21.3
+    Copyright (c) Donald Stufft and individual contributors.
+PyYAML 5.4.1 pypi:PyYAML/5.4.1
+    Copyright (c) 2017-2021 Ingy döt Net

```

Comment:
This dataclasses, and Pacakging copyright statements are underindented as compared to the other items; is this some kind of notation that it is manually maintained or just an inconsistency?

Reply:
Hard tabs vs soft tabs issue.

Reply:
Fixed!